packages/meditrak-server/src/routes/userEntityPermissions/assertUserEntityPermissionPermissions.js

```diff
@@ -23,6 +27,52 @@ export const assertUserEntityPermissionPermissions = async (
   return true;
 };
 
+export const assertUserEntityPermissionEditPermissions = async (
+  accessPolicy,
+  models,
+  userEntityPermissionId,
+  updatedFields,
+) => {
+  // Check we have permission to access the record we're trying to edit
+  await assertUserEntityPermissionPermissions(accessPolicy, models, userEntityPermissionId);
+  // Check we have permission for the changes
+  await assertUserEntityPermissionUpsertPermissions(accessPolicy, models, updatedFields);
+
+  // Final check to make sure we're not editing a BES admin access permission
+  // There's nothing we could edit that couldn't be abused
```

Comment:
```suggestion
  // which is the only thing we could edit that could be abused
```
I didn't quite get the comment - maybe it means this?